src/composables/ui.ts

```diff
@@ -12,11 +12,12 @@ export function useUi() {
     }
   }
 
-  function handleVisibilityChange(event: Event) {
+  function handleVisibilityChange(event: Partial<Event>) {
     isAppActive.value = !(event.target as Document).hidden;
   }
 
   function initVisibilityListeners() {
+    handleVisibilityChange({ target: document });
```

Comment:
I think there is no need to pass any arguments to the `handleVisibilityChange` function. As we are operating on the `document` object it could be directly used there this way:
```
isAppActive.value = !document.hidden;
```